test/c++11/tracer_test.cc

```diff
@@ -16,11 +16,10 @@ int main() {
     Dictionary val4 = { std::make_pair("hello", val3),
 			std::make_pair("whatever", val2) };
 
-    StartSpanOptions sopts;
-    sopts.operation_name = "span/test";
-    sopts.tags = std::move(val4);
-
-    auto span = Tracer::Global().StartSpanWithOptions(sopts);
+    auto span = Tracer::Global().StartSpan("span/untraced", {
+	StartTagOption("hello", std::move(val3)),
```

Comment:
This is an initializer list, and I'm not sure how else to indent it. The initializer_list machinery is intended to support syntax like:

int x[] = { 1, 2, 3 };

Now you can write

vector<int> x = { 1, 2, 3 };

But you can also place { a, b, c } in an expression context, which is what I've done. I'm still not sure how to indent it though. Let's use clang-format :)